packages/epinio/epinio/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,28 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,5 +1,16 @@
+ annotations:
+   artifacthub.io/license: Apache-2.0
++  catalog.cattle.io/auto-install: epinio-crd=match
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/display-name: Epinio
++  catalog.cattle.io/experimental: "true"
++  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.24.0-0'
```

Comment:
Looks like Rancher 2.6.8 supports K8s 1.24, so is this intentional? If this chart supports k8s 1.24 this should be updated to `'>= 1.20.0-0 < 1.25.0-0'`

Reply:
Done. Commits 48ff98d3c8a1d2ada31aed460aeec575db054b2d, 891495b445cbaf4ff1395c87d64c4ae230a672e7

Reply:
FYI. I ran our e2d STD-UI-Latest-Chrome [here](https://github.com/epinio/epinio-end-to-end-tests/actions/runs/3142231784/jobs/5106189904) using k3s version `v1.24.6-rc1+k3s1` and worked well. Just a couple of broken ones in configurations for an ui element that were tested locally successfully.
![image](https://user-images.githubusercontent.com/37271841/192765892-2f38d67d-51b6-49e2-a32f-b20448fa186f.png)

Also manually tested Epinio (released version 1.2.0) with Rancher Desktop on MAC using Kubernetes version 1.24.4 and did not foresee major issues:
 
<img width="916" alt="image" src="https://user-images.githubusercontent.com/37271841/192766057-817d30b1-af7c-4aeb-b0d6-c67d3e1218a9.png">